src/Command/GetGoMRIStatisticsCommand.php

```diff
@@ -27,29 +28,44 @@ class GetGoMRIStatisticsCommand extends Command
      */
     protected $entityManager;
 
+    /**
+     * The LogActionItemRepository Repository, for it's filtered count methods.
+     *
+     * @var LogActionItemRepository $logActionItemRepository
+     */
+    protected $logActionItemRepository;
+
     /**
      * Class constructor for dependency injection.
      *
      * @param EntityManagerInterface $entityManager A Doctrine EntityManager.
      */
     public function __construct(
         EntityManagerInterface $entityManager,
+        LogActionItemRepository $logActionItemRepository,
     ) {
         $this->entityManager = $entityManager;
+        $this->logActionItemRepository = $logActionItemRepository;
         parent::__construct();
     }
 
     protected function configure(): void
     {
         $this
             ->setDescription(self::$defaultDescription)
+            ->addArgument('start', InputArgument::OPTIONAL, 'Starting date for DL count')
+            ->addArgument('end', InputArgument::OPTIONAL, 'Ending date for DL count')
         ;
     }
 
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $io = new SymfonyStyle($input, $output);
 
+        $startEpoch = strtotime($input->getArgument('start'));
+        $endEpoch = strtotime($input->getArgument('end'));
```

Comment:
Why create a new `Datetime` here instead.

Reply:
I see, datetime with nothing becomes now. You could still do it, but just check if the argument is set.